Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN set -ex; \
     ; \
     rm -rf /var/lib/apt/lists/*
 
-ENV ALL_DOCKER_VERSIONS 1.3.3 1.4.1 1.5.0
+ENV ALL_DOCKER_VERSIONS 1.5.0
```

Comment:
We still want to test against older versions, but I think we can disable the tests that we expect to fail when the version is older.


If we accept api `version` from the environment then we can set it based on the version we're testing against.